### DISCUSSION TITLE
15 settembre 2022 - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#18] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/c3b8837b-e7a5-4025-b201-ecd878abc35b)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/f1eb6a96-f3e1-447a-a639-a783d940b5e1)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/c181b48f-b9c4-4a72-862c-93d7a8141bcd)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/747e962f-5457-433d-b923-772f037e1dfb)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/eb867d91-8d36-4529-b49f-81980d7d6aed)

**Reply by alemilos** [reply]
Anche CD è una chiave

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 17:43:41](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/05ec3664-af48-4338-8ecb-b5e5fcf6e5eb)
![IMAGE 2024-01-14 17:43:47](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/9d60d13a-3533-4d65-9320-3c31cfc090e6)

**Comment by daromacarlo** [comment]
punto 2a) ![Immagine 2024-02-04 215123](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/7bd55f2a-7fb2-46bb-8471-0e01bc76dc49)
